src/Recoil.rei

```diff
@@ -84,6 +101,16 @@ type value('value) = 'value;
 external useRecoilValue: t('value, 'mode) => value('value) =
   "useRecoilValue";
 
+type loadableValue('a) = {
+  state: State.t,
+  getValue: (. unit) => 'a,
```

Comment:
I had to add this `.` to avoid currying and getting transpiled to `Curry._1`. Do you have any better idea? It makes calling the function a bit strange to me especially since there are no arguments

Reply:
I wouldn't worry too much about the `Curry._1` there, having to use uncurry syntax would be a little annoying in terms of API.

Reply:
Problem is that the `Curry._1` generated an error at runtime (I don't why). Anyway, the `bs.send` I use know fixes it.